spec/requests/api/status_spec.rb

```diff
@@ -4,13 +4,29 @@
   let(:user) { create(:user) }
   let!(:project) { create(:project) }
   let!(:project_django) { create(:project, name: 'Django', platform: 'Pypi') }
+  let!(:repository) { create(:repository) }
+  let!(:maintenance_stat) { create(:repository_maintenance_stat, repository: repository)}
+
+  before do
+    user.current_api_key.update_attribute(:is_internal, true)
+
+    project.repository = repository
```

Comment:
Can you not create the repository first and then have the project factory take a repository parameter instead of setting `project.repository` here?

Reply:
It wasn't set up that way in `factories.rb` and I'm a little confused on the ownership of the models. The repository `has_many` projects and a Project belongs to a Repository.